examples/example_client_forum.py

```diff
@@ -0,0 +1,77 @@
+import socket
+import sys
+import requests
+
+ser_port = 9443 # server-router using 9443 instead of 443 to avoid sudo during test
+
+def jsonify(user, msg):
+    max_chars = 256
+    num_attributes = 2
+    max_chars -= (num_attributes*6+1) # "":"", and {} but last line has no comma
+    return "{\"user\":\"" + user + "\",\"msg\":\"" + msg + "\"}";
```

Comment:
Good catch. Maybe I will add something to scrub quotes from key names. If I do add any of the future changes I suggested, I will change this to pass in a dictionary instead of user and message. That sounds like it would actually work perfectly with the json.dumps which you suggested.